mne_bids/write.py

```diff
@@ -1483,6 +1499,21 @@ def write_raw_bids(raw, bids_path, events_data=None, event_id=None,
     mne.events_from_annotations
 
     """
+    if events_data is not None and events is not None:
+        raise ValueError('Only one of events and events_data can be passed.')
+
+    if events_data is not None:
+        warn(
+            message='The events_data parameter has been deprecated in favor '
+                    'the new events parameter, to ensure better consistency  '
+                    'with MNE-Python. The events_data parameter will be '
+                    'removed in a future release of MNE-BIDS. Please use the '
```

Comment:
When you deprecate say which version will drop support

Reply:
We need to decide upon that :)

Reply:
I'd say 0.14!